deploy/docker/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM registry.access.redhat.com/ubi7-minimal
+FROM registry.access.redhat.com/ubi7-minimal AS base-amd64
+FROM registry.access.redhat.com/ubi8-minimal AS base-arm64
+FROM registry.access.redhat.com/ubi8-minimal AS base-ppc64le
+
+FROM base-${TARGETARCH:-amd64}
```

Comment:
I noticed the var name here is "TARGETARCH" but the Makefile uses "TARGET_ARCH" (notice the underscore in the Makefile var). I personally would prefer them to all use the same thing - so for consistency, make them all one or the other (I would vote on making them all "TARGET_ARCH" unless there is specific reason why they are named as they are).

Reply:
TARGETARCH  is [platform arg](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope) and will automatic pass by docker. and this is important feature for `docker buildx build` to build mulit-arch images.